### ISSUE TITLE
[Feature Request] Button Hide to "Button" Options

### ISSUE THREAD

**Issue by boktai1000** [funkydude/BasicChatMods#40]
## Describe the problem
Give "Buttons Hide" more options, not only just hide but also button position, placing buttons inside the chat window, hiding unless mousover, etc. A few simple options here for customization without getting crazy, and keeping the addon simplistic yet offer a basic level of cusotmization.

### What version of the addon are you using? (Stating 'latest' is not useful)
v2-bcc
v5-classic

**Comment by funkydude** [comment]
Yup, this is something I've been thinking about as they add more and more buttons (like the TTS button on 9.1 PTR). I think just hiding them all is a bit of a dated concept.
